server/game/cards/11-TCaR/DevilsSixGun.js

```diff
@@ -0,0 +1,62 @@
+const GoodsCard = require('../../goodscard.js');
+const GameActions = require('../../GameActions/index.js');
+let selectedJoker = null;
+
+class DevilsSixGun extends GoodsCard {
```

Comment:
Probably a more long-term thing, but should we have a GadgetCard extend GoodsCard?

Asking now as that decision will probably be good to make earlier rather than later.

Reply:
We do not need `GadgetCard` as gadget is easily identified by the keyword and does not have any specific functions that should be put into the new class.